app/views/assignment_journals/index.html.slim

```diff
@@ -0,0 +1,42 @@
+- @assignment_journals ||= @assignment.assignment_journals
+
+.row
+  .col-xs-12
+    h1= t_title
+
+.row
+  .col-xs-12
+    table.table.table-striped
+      thead
+        tr
+          - if current_user.superadmin?
```

Comment:
Maybe you can check for a specific policy here instead of `superadmin?` directly? (works better if we change the policy later)

Reply:
I have no existing policy for AssignmentJournal that allows only superadmin. Should I create a controller method to authorize only for that?